src/bucket/BucketList.cpp

```diff
@@ -154,9 +154,9 @@ BucketLevel::prepare(Application& app, uint32_t currLedger,
         }
     }
 
-    mNextCurr =
-        FutureBucket(app, curr, snap, shadows, currLedgerProtocol,
-                     BucketList::keepDeadEntries(mLevel), countMergeEvents);
+    mNextCurr = FutureBucket(app, curr, snap, shadows, currLedgerProtocol,
```

Comment:
now BucketList::keepDeadEntries(mLevel) can be computed inside FutureBucket as well, so it is possible to get rid of one of the arguments here